source/agora/common/BlockStorage.d

```diff
@@ -221,6 +221,9 @@ public class BlockStorage : IBlockStorage
         Params:
             findex = the index of the file.
 
+        Returns:
+            Returns true if success, otherwise returns false.
+
```

Comment:
This looks good, but could you also rename the commit comment `"Fix document"` to `Add missing Return docs to BlockStorage`?